b2share/config.py

```diff
@@ -260,3 +260,5 @@
 PREFERRED_URL_SCHEME = 'http'
 
 FILES_REST_DEFAULT_MAX_FILE_SIZE = 1000 * 1024 * 1024 # 1 GB by default for RC4
+
+SITE_FUNCTION = 'testing'
```

Comment:
I don't see any other place where `SITE_FUNCTION`. It seems that this would mark the docker container as `testing` too. @llehtine will then need to override this when deploying in production.